Blish HUD/GameServices/ContentService.cs

```diff
@@ -108,6 +109,12 @@ public enum FontStyle {
 
         public Microsoft.Xna.Framework.Content.ContentManager ContentManager => Blish_HUD.BlishHud.ActiveContentManager;
 
+        private ICacheMethod _sharedRenderServiceCache;
+
+        public ICacheMethod GetRenderServiceCacheMethod() {
+            return _sharedRenderServiceCache ?? (_sharedRenderServiceCache = new MemoryCacheMethod());
```

Comment:
Why don't we use the ArchiveCacheMethod here?

Reply:
I've had a mixed experience with `ArchiveCacheMethod` in the past and troubles getting it to flush to the disk correctly causing it to not keep a cache of things requested that session until the session ends and it flushes to disk.  This causes all requests to get re-requested every request.  There were recent changes to it after some other folks brought up having similar issues, so I'll need to review those.

Ultimately, I need more time to review the current implementation in Gw2Sharp to ensure it works correctly when we use it and will have a PR to update this to an `ArchiveCacheMethod` once I have done that.  Using the `MemoryCacheMethod` for now will at least cover caching for the active session until that work is done.